autogtp/Production.cpp

```diff
@@ -178,16 +182,22 @@ void Production::getResult(const QString& file, float duration) {
 }
 
 void  Production::printTimingInfo(float duration) {
+    if (m_movesMade == 0 || m_gamesPlayed == 0) {
+        return;
+    }
     auto game_end = std::chrono::high_resolution_clock::now();
     auto total_time_s =
         std::chrono::duration_cast<std::chrono::seconds>(game_end - m_start);
     auto total_time_min =
         std::chrono::duration_cast<std::chrono::minutes>(total_time_s);
+    auto total_time_millis =
+        std::chrono::duration_cast<std::chrono::milliseconds>(total_time_s);
```

Comment:
I should really use (game_end - m_start) to calculate total_time_millis then cast that to total_time_s and total_time_min but the net effect is in the 4 or 5th sig fig.